lib/src/components/NxTag/NxTag.tsx

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { FunctionComponent } from 'react';
+import classnames from 'classnames';
+import { faPlusCircle } from '@fortawesome/free-solid-svg-icons';
+
+import './NxTag.scss';
+import NxFontAwesomeIcon from '../NxFontAwesomeIcon/NxFontAwesomeIcon';
+import NxOverflowTooltip from '../NxTooltip/NxOverflowTooltip';
+import { Props, propTypes } from './types';
+export { Props } from './types';
+
+const NxTag: FunctionComponent<Props> =
+    function NxTag(props) {
+      const { children, className, tagColor, ...attrs } = props,
+          tagClasses = classnames('nx-tag', className, {
+            [`nx-tag--${tagColor}`]: tagColor,
+            ['nx-tag--default']: !tagColor
+          });
+
+      return (
+        <div className={tagClasses} {...attrs}>
+          <NxOverflowTooltip>
+            <div className="nx-tag__text">{children}</div>
+          </NxOverflowTooltip>
+        </div>
+      );
+    };
+
+NxTag.propTypes = propTypes;
+export default NxTag;
+
+export const NxSelectableTag: FunctionComponent<Props> =
```

Comment:
I don't think we ever really put multiple top-level public components in the same file like this.  I'd move this to a separate file, though having it in the same directory is fine.

Reply:
> I don't think we ever really put multiple top-level public components in the same file like this. I'd move this to a separate file, though having it in the same directory is fine.

I copied this directly from `NxAlert`

Reply:
> I copied this directly from NxAlert

Ah good point